docsets/Python_zh_cn/docset.json

```diff
@@ -0,0 +1,27 @@
+{
+    "name": "Python (Chinese)",
+    "version": "3.7.4",
+    "archive": "Python_zh_cn.tgz",
+    "author": {
+        "name": "bokix",
+        "link": "https://github.com/bokix"
+    },
+    "aliases": ["python", 
+                "chinese",
+                "cn",
+                "zh"],
+
+    "specific_versions": [
+        {
+            "_comment": "API document for python2 and python3"
+        },
+        { 
+            "version": "3.7.4",
+            "archive": "versions/3.7.4/Python_zh_cn.tgz",
```

Comment:
This path does not exist. The path you have is: `versions/3.7.4.1/Python_zh_cn.tgz`.

Reply:
Sorry! but in Mac Finder, the folder name is "3.7.4" and in Terminal is "3.7.4.1" , I don't know why. Anyway, I modified the name of the folder.
<img width="849" alt="2019-09-02 09 44 12" src="https://user-images.githubusercontent.com/3327265/64085889-cc095500-cd67-11e9-844f-a62ca1bfa40e.png">